### ISSUE TITLE
Insert form submission directly into Nextcloud Deck Card

### ISSUE THREAD

**Issue by elephantastyczny** [nextcloud/forms#1850]
**Nextcloud (please complete the following information):**
- Nextcloud-Version: [28.0.0 +]
- Forms-Version: [4.0.0 +]

**Describe the solution you'd like**

Id like to see Forms to be also used as a "ticketing system".
We should be able to predefine, i.e. in dropdown type question, who is responsible for dealing with the issue and once from is submitted:

- new card is created in predefined deck and list
- correct person assigned and notified (predefined in question)
- correct tag is selected (predefined in question)
- card description copied from predefined title question (short text)
- other submitted data copied into card notes

**Comment by Chartman123** [comment]
As this is really an edge case to use Forms, I don't see it coming any time soon. There are lots of ticketing systems available already and so I also don't think that we should abuse two apps designed for something else.

@susnux what do you think?

**Comment by elephantastyczny** [comment]
I was rather thinking about internal solution for groups.

Imagine 100 employees pool using just Nextcloud and they would like to have a single point of reporting issues:



*	Ordering supplies IT/non IT
*	general maintenance (lights, faulty locks)
*	HR related items onboarding / offboarding / change job requests

 

I believe this could replace any 3rd party solution 😊

No pressure, just saying 😊

**Comment by susnux** [comment]
Yes we can keep this open but with very low priority, we should probably first think about:
- #805 
- #1748

**Comment by CelianGdfrd** [comment]
Hi, is there any advancement on this, please ? It would be incredibly useful for associations I work with

**Comment by Chartman123** [comment]
I don't know why I didn't notice it earlier, but this is duplicate of  #1772
